tests/backends/sqlite/tests.py

```diff
@@ -19,6 +23,14 @@
 class Tests(TestCase):
     longMessage = True
 
+    def test_min_version(self):
+        msg = 'SQLite 3.8.3 or later is required (found 3.8.2).'
+        with mock.patch.object(dbapi2, 'sqlite_version_info', (3, 8, 2)), \
+                mock.patch.object(dbapi2, 'sqlite_version', '3.8.2'), \
+                self.assertRaisesMessage(ImproperlyConfigured, msg):
+            import django.db.backends.sqlite3.base
+            importlib.reload(django.db.backends.sqlite3.base)
```

Comment:
I'm a bit worried about the side effect of reloading this module given how much stuff depend on it indirectly. Tests are passing which is a good sign but that'll make yet another global side effect to think about when debugging weird issues with the test suite.

I don't have any other idea of how to test this in another way but maybe raising the error on `DatabaseWrapper.__init__` instead could bring the same benefit while being easier to test?

FWIW [we haven't tested such errors](https://github.com/django/django/blob/88619e6129fd8c6668242f1acc1465ca175d2948/django/db/backends/mysql/base.py#L35-L36) in the past and I would be ok with doing the same here.

Reply:
Or add a function which can be tested and call it at the module level?

Reply:
yeah that'd work as well, good idea!